src/test/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/handlers/presubmit/InternalAdaGenerateHearingBundleHandlerTest.java

```diff
@@ -0,0 +1,169 @@
+package uk.gov.hmcts.reform.iacasedocumentsapi.domain.handlers.presubmit;
```

Comment:
Can we please use the letter package for all the notification related letter generation?